simple-examples/package.json

```diff
@@ -14,8 +14,7 @@
     "url": "https://github.com/hatchet-dev/hatchet-typescrpt-quickstart.git"
   },
   "scripts": {
-    "tsc:build": "tsc && resolve-tspaths",
-    "generate": "./generate-protoc.sh",
```

Comment:
how are we handling this generation now?

Reply:
this seems just like a leftover from a copy paste, I don't think we need to generate anything here do we?

Reply:
you're correct, I didn't realize we were in the quickstart -- brain was in the wrong context.